src/utils/validation.ts

```diff
@@ -104,13 +104,13 @@ function validateElement(
     };
     validations?: ResolvedCustomValidation[];
   },
-  repeat: number
+  repeat: number | undefined
 ): string {
   // First priority is standard validations for servar fields
   const { servar, validations } = element;
   if (servar) {
     let fieldVal: any = fieldValues[servar.key];
-    if (servar.repeated) fieldVal = fieldVal[repeat];
+    if (servar.repeated && repeat !== undefined) fieldVal = fieldVal[repeat];
```

Comment:
do we need both checks? in what case is servar.repeated true but repeat is undefined

Reply:
Ok, not generally the way I like to code, but made the change.